oeqLinkMigrationLTI/src/main/java/org/apereo/openequella/integration/blackboard/linkmigrationlti/StubFixer.java

```diff
@@ -0,0 +1,64 @@
+package org.apereo.openequella.integration.blackboard.linkmigrationlti;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class StubFixer extends AbstractFixer
+{
+	private final List<String> content = new ArrayList<String>();
+
+	public StubFixer()
+	{
+		super();
+	}
+
+	public void addContent(String newContent)
+	{
+		content.add(newContent);
+	}
+
+	@Override
+	public void submit(HttpServletRequest request) throws Exception
+	{
+		for( String c : content )
+		{
+			logMessage(0, "OLD " + c);
+			FixTextFeedback feedback = new FixTextFeedback();
+			logMessage(1, "NEW " + fixText(c, feedback));
+			if( feedback.fixedHardCodedIds )
+			{
+				logMessage(1, "Hard coded ID fixed");
+			}
+			if( feedback.fixedNonTokenedUrls )
+			{
+				logMessage(1, "Non token URL fixed");
+			}
+		}
+		System.out.print(log.toString());
+	}
+
+	@Override
+	protected int getBlackboardVersion()
+	{
+		return 8;
```

Comment:
The other method for the version was dynamic.  Should this use the other method instead of hardcoded?

Reply:
Not anymore in the code